lib/active_fedora/nested_attributes.rb

```diff
@@ -181,8 +183,8 @@ def raise_nested_attributes_record_not_found(association_name, record_id)
       raise ObjectNotFoundError, "Couldn't find #{reflection.klass.name} with ID=#{record_id} for #{self.class.name} with ID=#{id}"
     end
 
-    def call_reject_if(association_name, attributes)
-      return false if has_destroy_flag?(attributes)
+    def call_reject_if(association_name, attributes, allow_destroy)
```

Comment:
If they pass destroy, we should not reject it here. We check for that case in `assign_to_or_mark_for_destruction`